src/api/controllers/idea.ts

```diff
@@ -1,6 +1,7 @@
 import camelcaseKeys from 'camelcase-keys';
 import {FastifyRequest} from 'fastify';
 import {Controller, GET, POST, PUT} from 'fastify-decorators';
+import {snakeCase} from 'lodash';
```

Comment:
import lodash dibuat treeshake yof. klo gak besar banget size lodash yg dipake.
`import snakeCase from 'lodash/snakeCase`

Reply:
hmm ini sih di server rie, tapi di bundling juga sih be nya